src/main/java/com/minecolonies/coremod/entity/citizen/EntityCitizen.java

```diff
@@ -1436,6 +1435,7 @@ public void callForHelp(final Entity attacker, final int guardHelpRange)
 
         long guardDistance = guardHelpRange;
         AbstractEntityCitizen guard = null;
+        AbstractEntityCitizen secondGuard = null;
```

Comment:
Potentially this should be more generic. Make it a set or a list, etc. So that this can be extended if needed.

Reply:
hm I wanted to avoid having to sort a complete list and compare it all again

Reply:
could make it a map citizen id to Entity for quick access

Reply:
I'm sorting them by distance here, don't think a map would help with that :D

Reply:
That should be relatively simple. Make it an ordered list by distance such that you can always just poll the last one out each round. That would probably make even the existing one easier.